### DISCUSSION TITLE
bookmarklet question

### DISCUSSION THREAD

**Discussion by Theblackmamba824** [3kh0/ext-remover#234] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
is there a way that you can make a bookmarklet that can let you install blocked extensions

**Comment by ILikeBypassing** [comment, marked as answer]
not possible bc you cant install blocked extensions using any method since the administator blocked it
but if u want to see allowed extensions just go into chrome://policy and click show more on ExtensionInstallAllowlist and youll see the ids of all allowed extensions which u can install manually

**Reply by Theblackmamba824** [reply]
thx
